docs/content/troubleshooting/log_status_collect.md

```diff
@@ -1,41 +1,101 @@
 
-To collect and display status and logs related to the different components of IBM® block storage CSI driver, use the following Kubernetes commands and procedures:
+## Detecting errors
 
-## Status collection for CSI pods, daemonset, and statefulset
+To help pinpoint potential causes for stateful pod failure:
 
-`kubectl get all -n <namespace>  -l csi`
+1.  Verify that all CSI pods are running.
+    ```
+    kubectl get pods -n <namespace> -l product=ibm-block-csi-driver
+    ```
 
-### Log collection for IBM block storage CSI driver controller
+2.  If a pod is not in a _Running_ state, run the following command:
+    ```
+    kubectl describe -n <namespace> pod/<pod-name>
+    ```
+    View the logs.
 
-`kubectl log -f -n <namespace> ibm-block-csi-controller-0 -c ibm-block-csi-controller`
 
-### Log collection for IBM block storage CSI driver node (per worker node or PODID)
+## Status and log collection
+To collect and display status and logs related to the different components of IBM® block storage CSI driver, use the Kubernetes commands that are found in this section.
 
-`kubectl log -f -n <namespace> ibm-block-csi-node-<PODID> -c ibm-block-csi-node`
+Before you begin collecting logs, create a directory for the logs on the control-plane or bastion server.
 
-## Log collection for Operator for IBM block storage CSI driver
+For example:
 
-`kubectl log -f -n <namespace> ibm-block-csi-operator-<PODID> -c ibm-block-csi-operator`
+```
+mkdir logs
+```
+Save logs and status reports directly to the created directory by adding in the following string at the end of the collection command: `> logs/<log_filename>`.
 
-## Detecting errors
+**Important:** Be sure that the logs cover any relevant time frames for the specific issues that you are trying to debug when gathering logs from the storage system.
 
-To help pinpoint potential causes for stateful pod failure:
+**Note:** All commands here are listed with the collection command with the `logs` folder name example. Change the folder name according as needed.
 
-1.  Verify that all CSI pods are running.
+### General status and log collection for IBM Support
+Be sure to run the following steps and copy the output to an external file, when engaging IBM Support and sending log collections.
 
-    ```
-    kubectl get pods -n <namespace> -l csi
-    ```
+1. Check the node status.
+    
+    `kubectl get nodes`
+2. Check the CSI driver component status.
 
-2.  If a pod is not in a _Running_ state, run the following command:
+    `kubectl get -all-namespaces pod -o wide | grep ibm-block-csi`
+3. Check if the PersistentVolumeClaims (PVCs) are _Bound_.
 
-    ```
-    kubectl describe -n <namespace> pod/<pod-name>
-    ```
+    `kubectl get -n <pvcs_namespace> pvc -o=jsonpath='{.items[?(@.metadata.annotations.volume\.beta\.kubernetes\.io/storage-provisioner=="block.csi.ibm.com")].metadata.name}'`
```

Comment:
@rivkap please change the command to:
`kubectl get -n default pvc -o=jsonpath='{range .items[?(@.metadata.annotations.volume\.beta\.kubernetes\.io/storage-provisioner=="block.csi.ibm.com")]}{"PVC NAME: "}{@.metadata.name}{" PVC STATUS: "}{@.status.phase}{"\n"}{end}'`

It will give an example output:
`PVC NAME: demo-pvc-file-system PVC STATUS: Bound`

Reply:
LGTM

Reply:
done. thanks @roysahar